code/index.html

```diff
@@ -1,30 +1,178 @@
+</head>
+<body>
+    
+    <header class="hero-section">
+      <div class="hero-video">
+        <video src="./videos/video_large.mp4" autoplay muted loop></video>
+        <img src="./videos/poster1.png" title="Your browser does not support the video tag.">
       </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="hero-text">
+        <h1>Two Fish</h1>
+        <h2>Contest</h2>
+        <h3>Win a scuba-diving trip for you and your special-one!</h3>
+        <a href="#intro" class="btn">Register now</a>
       </div>
+    </header>
+
+    <section id="intro">
+      <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+      Vitae error ratione a dolorem odio sit omnis, iure est optio sunt nulla.
+      Vitae error ratione a dolorem.</p>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <section class= flex-parent>    
+
+      <!--    FORM -->
+
+      <div class= "flex-child form">
+        <h2>Registration</h2>
+        <form 
+            class="form-container" 
+            action="http://httpbin.org/anything" 
+            method="POST">
+            
+            <div class="form-group">
+            <label for="firstName"></label>
+            <input 
+            id="firstName" 
+            type="text" 
+            placeholder="First name *"
+            name="firstNameValue"
+            required>  
+            </div>
+            
+            <div class="form-group">
+            <label for="lastName"></label>
+            <input 
+            id="lastName" 
+            type="text"
+            placeholder="Last name *"
+            name="lastNameValue"
+            required> 
+            </div>
+    
+            <div class="form-group">
+            <label for="telephoneNumber"></label>
+            <input 
+            id="telephoneNumber" 
+            type="tel" 
+            placeholder="Telephone number"
+            name="telephoneNumberValue">
+            </div>
+            
+            <div class="form-group">
+            <label for="email"></label>
+            <input 
+            id="email" 
+            type="email"
+            placeholder="E-mail*"
+            name="emailValue"
+            required>
+            </div>
+
+            <div class="form-group">
+            <label class="checkbox-label">
+            <input 
+            type="checkbox">
+            <span class="checkbox-custom rectangular"></span>
+            </label>
+            <div class="input-title">I accept the terms and conditions</div>
+            </div>
+
+            <div>
+            <button 
+            class="form-submit" 
+            type="submit">Submit
+            </button>
+            </div>   
+          </div>
+        </form>
+    
+      </div> 
+
+
+    <!-- ACCORDION -->
+    <div class="flex-child accordion"><h2>FAQ</h2>
```

Comment:
Nested elements should be given newlines (and be indented)

Reply:
Done